src/include/btmem.h

```diff
@@ -1028,14 +1028,13 @@ struct __wt_ref {
         WT_UPDATE **update;   /* Page instantiated, update list for subsequent commit/abort */
     } ft_info;
 
+#ifdef HAVE_REF_TRACK
 /*
- * In DIAGNOSTIC mode we overwrite the WT_REF on free to force failures. Don't clear the history in
- * that case.
+ * If ref tracking is enabled, we overwrite the WT_REF on free to force failures. Don't clear the
```

Comment:
I think this comment should still say DIAGNOSTIC mode. I don't see anything in this diff that indicates `HAVE_REF_TRACK` will zero out freed WT_REF memory.

Reply:
I don't follow this comment. Now the ref track structure is only enabled with HAVE_REF_TRACK not DIAGNOSTIC flag. It can be enabled in production build as well.